apps/dashboard/src/components/auth/auth-side-banner.tsx

```diff
@@ -0,0 +1,50 @@
+import { CircleCheck } from '../icons/circle-check';
+import { ShieldZap } from '../icons/shield-zap';
+import { Plug } from '../icons/plug';
+import { Sparkling } from '../icons/sparkling';
+import { AuthFeatureRow } from './auth-feature-row';
+import { TrustedCompanies } from './trusted-companies';
+
+export function AuthSideBanner() {
+  return (
+    <div className="inline-flex w-full max-w-[476px] flex-col items-center justify-center gap-[50px] p-5">
+      <div className="flex flex-col items-start justify-start gap-4">
+        <div className="inline-flex items-center justify-start gap-3">
```

Comment:
Is there a recommended way of working with brand spacing? @LetItRock

Reply:
It's preferred to use built-in Tailwind classes as you've already done, but it's also fine to use the `-[{whatever}]` escape hatch syntax in cases where it makes sense to provide a better UX + look/feel